config/ldes-delta-pusher/ldes-instances.ts

```diff
@@ -66,9 +66,16 @@ export const ldesInstances = {
           "http://purl.org/dc/terms/modified",
         ],
       },
-      "http://www.w3.org/ns/activitystreams#Tombstone": [
-        "http://purl.org/dc/terms/modified",
-      ],
+      "http://www.w3.org/ns/activitystreams#Tombstone": {
+        healingPredicates: ["http://purl.org/dc/terms/modified"],
+        healingFilter: `FILTER NOT EXISTS {
+            GRAPH ?h {
+             ?s a ?otherType.
+             FILTER(?otherType != <http://www.w3.org/ns/activitystreams#Tombstone>)
+            }
+            ?h <http://mu.semte.ch/vocabularies/ext/ownedBy> ?org.
```

Comment:
pseudo explanation: Heal the entities the "active" entities, so possible tombstone but still used in the app